kahuna/public/js/errors/global.html

```diff
@@ -21,6 +26,11 @@
                target="_self" class="coloured-link">login again</a>
             to reauthenticate.
         </div>
+
+        <p>
+            If you can see this icon <img src="/assets/images/blocked-cookies.png" alt="blocked third party cookies" class="side-padded"/> in the address bar,
+            please follow <a href="https://docs.google.com/a/guardian.co.uk/document/d/1qlQhHM2NX3xWBIREB2O8D7S4ZtN0sYb7IDV2leE4Ubc/edit?usp=sharing" class="coloured-link" target="_blank">these instructions</a>.
```

Comment:
> Could you request it to be public for all instead?

@jamesgorrie I think you're the owner of the doc, can you change the permissions? I can't work out how to request access level changes from within Google Docs :cry: